src/components/accordion/header/styled.tsx

```diff
@@ -0,0 +1,166 @@
+import styled, { css } from 'styled-components';
+import { theme, Colour } from '@fellesdatakatalog/theme';
+import Icon from '../../icon';
+
+interface Props {
+  $collapsed?: boolean;
+  $showAlert: boolean;
+}
+
+const Accordion = styled.div<Props>`
+  display: flex;
+  flex-direction: column;
+
+  min-height: ${props =>
+    props.$collapsed ? theme.spacing('S96') : theme.spacing('S72')};
+  border-radius: ${theme.spacing('S4')};
+
+  :hover {
+    background-color: ${theme.colour(Colour.NEUTRAL, 'N70')};
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6,
+    p {
+      color: ${theme.colour(Colour.NEUTRAL, 'N0')};
+    }
+
+    span {
+      background-color: ${theme.colour(Colour.NEUTRAL, 'N0')};
+      color: ${theme.colour(Colour.NEUTRAL, 'N70')};
+    }
+
+    svg {
+      & * {
+        stroke: ${theme.colour(Colour.NEUTRAL, 'N0')};
+      }
+    }
+  }
+
+  transition: 0.5s;
+  * > {
+    transition: 0.5s;
+  }
+
+  ${props =>
+    props.$showAlert &&
+    css`
+      background-color: ${theme.colour(Colour.RED, 'R30')};
+    `}
+`;
+
+const CollapseButton = styled.button`
+  display: flex;
+  flex-direction: column;
+  background-color: transparent;
+  font-family: inherit;
+  border: none;
+  padding: 0px ${theme.spacing('S24')};
+  cursor: pointer;
+`;
+
+const DivHeading = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: ${theme.spacing('S12')};
+  margin: ${theme.spacing('S24')} 0px 0px;
+`;
+
+const AccordionTitleContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: ${theme.spacing('S12')};
+  flex: 1;
+`;
+
+const AccordionTitle = styled.h2<Props>`
+  display: flex;
+  justify-self: flex-start;
+  align-items: baseline;
+
+  font-weight: ${theme.fontWeight('FW700')};
+  font-size: ${theme.fontSize('FS24')};
+  line-height: ${theme.spacing('S32')};
+
+  padding: 0px;
```

Comment:
Er det kanskje unødvendig med unit her?

```suggestion
  padding: 0;
```